spec_01.md

```diff
@@ -0,0 +1,46 @@
+- Spec: 1
+- Title: Methods on libraries
+- Version: 0.1
+- Last-Modified: 2019-07-19
+- Authors: [James Bednar](),
+           [Julia Signell](jsignell@gmail.com),
+           [Jake Vanderplas]()
+- Status: Active
+- Type: Standards
+- Content-Type: text/markdown
+- Created: 2019-07-19
+
+## Abstract
+This spec describes a proposed shared API for enabling plotting libraries.
+
+This spec uses LIBRARY to refer to any specific visualization library.
+
+## If a method doesn't make sense
+If a given operation doesn't make sense for that library, then it can satisfy the spec by simply having that method return a message to that effect ("JSON support not available in LIBRARY").
```

Comment:
We should probably make this text be computer parseable so that the test suite can be written in a library-agnostic way.  E.g.:

```suggestion
If a given operation doesn't make sense for that library, then it can satisfy the spec by simply having that method return a message to that effect ("Unsupported: LIBRARY does not provide JSON output").
```

(Where the test suite would look for "^Unsupported:" and consider that a successful output.)